hazelcast/src/main/java/com/hazelcast/map/impl/operation/MultipleEntryOperation.java

```diff
@@ -104,7 +104,12 @@ public int getAsyncBackupCount() {
     @Override
     public Operation getBackupOperation() {
         EntryBackupProcessor backupProcessor = entryProcessor.getBackupProcessor();
-        return backupProcessor != null ? new MultipleEntryBackupOperation(name, keys, backupProcessor) : null;
+        MultipleEntryBackupOperation backupOperation = null;
+        if (backupProcessor != null) {
+            backupOperation = new MultipleEntryBackupOperation(name, keys, backupProcessor);
+            backupOperation.setWanEventList(wanEventList);
```

Comment:
I first didn't see where the `wanEventList` entries were created (since the `new WanEventWrapper` is gone), but then I realized they were just copied. Makes sense and looks good to me.